classes/widgets.php

```diff
@@ -284,29 +284,29 @@ public function form( $instance ) {
 				<input type="checkbox" class="checkbox" id="<?php echo $this->get_field_id( $show_field ); ?>"
 				       name="<?php echo $this->get_field_name( $show_field ); ?>"<?php checked( $show ); ?> />
 				<label
-					for="<?php echo $this->get_field_id( $show_field ); ?>"><?php _e( 'Show ' . $tax_info['plural'], 'simplemap' ); ?></label>
+					for="<?php echo $this->get_field_id( $show_field ); ?>"><?php _e( 'Show ' . $tax_info['plural'], 'SimpleMap' ); ?></label>
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 142 characters

<!-- Comment by [SideCI](https://sideci.com) -->